source/guides/guides/parallelization.md

```diff
@@ -151,18 +153,26 @@ For multiple runs to be grouped into a single run, it is required for CI machine
 
 You can test your application against different browsers and view the results under a single run within the Dashboard. Below, we simple name our groups the same name as the browser being tested:
 
-- The first group can be called `electron`. *Electron is the default browser used in Cypress runs*.
+- The first group can be called `Windows/Chrome 69`. 
+
+  ```shell
+  cypress run --record --group Windows/Chrome-69
+  ```
+
+- The second group can be called `Mac/Chrome 70`. 
 
   ```shell
-  cypress run --record --group electron
+  cypress run --record --group chrome --browser Mac/Chrome-70
   ```
 
-- The second group can be called `chrome`.
+- The third group can be called `Linux/Electron`. *Electron is the default browser used in Cypress runs*.
 
   ```shell
-  cypress run --record --group chrome --browser chrome
+  cypress run --record --group chrome --browser Linux/Electron
```

Comment:
See above

Reply:
Actually - since Electron is the default browser, this one would not use the browser flag. Instead the `--browser chrome` flag should be added to the Windows example above.